tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/AiCompletion/AdditionalInfo.cs

```diff
@@ -0,0 +1,23 @@
+using System.Text.Json.Serialization;
+using System.ComponentModel.DataAnnotations;
+
+namespace Azure.Sdk.Tools.Cli.Models.AiCompletion
```

Comment:
The name `AiCompletion` might be too general. Just trying to understand is it the models to build the request to call the Azure SDK knowledge base service?  Might be worth to rename it.

Reply:
I will change to AzureSDKKnowledgeCompletionAPI